### ISSUE TITLE
Certfied or Evaluated?

### ISSUE THREAD

**Issue by gautierchomel** [w3c/publ-a11y#473]
While trying to explain the difficulties of translating “Certified” into French (#472), I realized that [EPUB Accessibility 1.1 section 3.5 Conformance reporting](https://www.w3.org/TR/epub-a11y-11/#sec-conf-reporting) uses the wording “Evaluator” and “Evaluated.” 

I suggest that we adopt this wording for the Display guide instead of “Certified By” and “Certified”. 

I think it best reflects the reality, especifically in the case of Self evaluation. 

It also allows for better visibility of third party certifications, which includes more than just an evaluation.

**Comment by GeorgeKerscher** [comment]
Do we need these words added to the terms section?

**Comment by mattgarrish** [comment]
I recall we had this problem of certification vs. evaluation when we went through the ISO process. They also view certification in legal terms, so the language of certification was changed to evaluation. It was too late to change the property names, but if you compare the IDPF 1.0 version to the 1.1 version there's now a distinct lack of use of certify beyond the those names.

**Comment by GeorgeKerscher** [comment]
I thought I would add this post from a NISO discussion list in the library space:
**
I think there's a critical issue to discuss, along with the risk it poses. 
•	The critical issue is recognizing a conflict of interest in self-declarations of accessibility. 
•	The risk is a breach of trust with library users who depend on accessibility metadata.
We would recognize the conflict of interest in a restaurant self-declaring its kitchen met health standards. That's why there are restaurant inspectors.  Likewise, there's also a conflict of interest in publishers or libraries self-declaring their content was built or remediated to accessibility standards.  
 
What's the risk?  Outrage from users, betrayal of trust, and harm to users. 

In the context of websites, there has been tremendous vocal outrage within blind communities about companies that make false statements about creating accessible websites.  In 2021, the US National Federation of the Blind (NFB) issued a resolution [Regarding the Use of Overlays to Make Websites Accessible to the Blind](https://nfb.org/resources/speeches-and-reports/resolutions/2021-resolutions#04).

In this 2021 [personal appeal](https://www.youtube.com/watch?v=R12Z1Sp-u4U) on YouTube (3.5 minutes), Haben Girma, who is a Harvard law graduate and is Deafblind, expresses deep frustration about companies claiming they can easily make websites accessible.  She emphatically expresses how trust has been broken.

Two years later, the founder of one of the companies issued [an apology](https://nfb.org/images/nfb/publications/bm/bm23/bm2305/bm230503.htm) to the NFB. At the 2023 convention, the founder also [addressed](https://nfb.org/images/nfb/publications/bm/bm24/bm2406/bm240618.htm) NFB members about how the company was trying to rebuild trust. 

In the context of libraries and books, I asked a blind researcher to join a discussion with sighted librarians about accessible discovery experiences.  On the issue of books being described as accessible when they weren't, the researcher said it was like being gaslighted. 

What's a solution for this IFLA Network? A solution might include recognizing the need for independent 3rd parties to certify claims related to accessible metadata.  In the context of publishing, we find an example in Benetech, which certifies the workflow of publishers wanting to create accessible books.   Also, encouraging publishers and libraries to engage and work with people with disabilities, including in co-designing enjoyable reading experiences, could help.

**Comment by gautierchomel** [comment]
Could we imagine using "Evaluated" when there is no certifier credential and "{certifier credential} certified " when there is a certifier credential?

**Comment by clapierre** [comment]
The editors met and we decided to handle this by creating two sets of strings one for a stricter **"Certification"** vs. an **"Evaluation"**

We will also add notes to the main guidelines document and both techniques documents and comments in the JSON file stating that it will be up to the implementor to decide if certification vs. evaluation is warranted for their country's interpretation of these terms.
 
conformance-**certifier** = "**certifier**"
conformance-**certifier**-credentials = " with a **credential** of "
conformance-**certification**-info = "The publication was **certified** "
conformance-**certifier**-report = "For more information refer to the **certifier's** report"

and

conformance-**evaluator** "**evaluator**"
conformance-**evaluator**-credentials = " with a **reference** of "
conformance-**evaluation**-info = "The publication was **evaluated** "
conformance-**evaluation**-report = "For more information refer to the **evaluator's** report"


Not really sure if we need the two prefix's unless we change the ID's from
conformance-**certifier**-credentials-prefix = " by"
conformance-**certification**-date-prefix = " on "
conformance-**evaluator**-credentials-prefix = " by"
conformance-**evaluation**-date-prefix = "on"
to
conformance-credentials-prefix = " by "
conformance-date-prefix = "on "

**Comment by GeorgeKerscher** [comment]
I believe we are getting away from requiring  a prefix and constructing a sentence. Instead we are making each statement a stand alone.

**Comment by clapierre** [comment]
we have discussed this and is good.

**Comment by clapierre** [comment]
We need to have a option for display strings for evaluators

**Comment by gregoriopellegrino** [comment]
@clapierre @mattgarrish where do we add them? In the principles with an example?

**Comment by clapierre** [comment]
I think so, that makes the most sense.

**Comment by gregoriopellegrino** [comment]
@mattgarrish @gautierchomel @GeorgeKerscher @clapierre I realized that there was this outstanding issue left over

**Comment by gautierchomel** [comment]
My understanding is that our solution will not change the existing but add an option to permute the four "certifier" values for "evaluator"
* [ID: conformance-certifier]
* [ID: conformance-certifier-credentials]
* [ID: conformance-details-certification-info]
* [ID: conformance-details-certifier-report]

That means adding an explanatory paragraph to each of the three documents and 4 "evaluator" values to the JSON. 

As it is, the vocabulary will depend exclusively on the localisation file, leaving the implementor with no choice.

**Comment by gregoriopellegrino** [comment]
Okay, so nothing for this version, we will discuss it later.
